composer.json

```diff
@@ -79,33 +81,27 @@
         "drupal/recaptcha_v3": "^2.0",
         "drupal/redirect": "^1.7",
         "drupal/scheduler": "^1.3",
+        "drupal/seven": "^1.0",
         "drupal/simple_sitemap": "^4.1",
         "drupal/switch_page_theme": "^4.0",
         "drupal/twig_extensions": "2.x-dev",
         "drupal/upgrade_status": "^4.0",
         "drupal/video_embed_field": "^2.4",
         "drupal/viewsreference": "^2.0@beta",
         "drupal/webform": "^6.2",
+        "drush/drush": "^12",
         "oomphinc/composer-installers-extender": "^2.0",
-        "twig/extensions": "^1.5"
+        "twig/extra-bundle": "^3.0"
     },
     "require-dev": {
-        "amazeeio/drupal_integrations": "^0.3.3",
-        "behat/mink-extension": "^2.2",
-        "drupal/core-dev": "^9.5",
-        "drupal/drupal-extension": "^4",
-        "drush/drush": "^11",
-        "mglaman/drupal-check": "^1.4",
-        "phpcompatibility/php-compatibility": "dev-develop",
-        "phpmd/phpmd": "^2.4"
+        "mglaman/drupal-check": "^1.4"
```

Comment:
Want to make sure these removals are good for everyone, particularly amazee, and the drupal. I don't have any particular opinions, but thought it's worth pointing out?

cc @froboy

Reply:
Looks like drush is a regular requirement instead of a dev req, but I think the others are removed.

Reply:
https://github.com/amazeeio/drupal-integrations seems like it very much should still be included. @bmartinez287 was this intentionally removed?